.gitmodules

```diff
@@ -15,7 +15,7 @@
 	branch = master
 [submodule "extern/topmodel/topmodel"]
 	path = extern/topmodel/topmodel
-	url = git@github.com:NOAA-OWP/topmodel.git
+	url = https://github.com/NOAA-OWP/topmodel.git
```

Comment:
Eventually we will get all these fixed, I'm pretty sure I have this in #304 as well.